app.js

```diff
@@ -0,0 +1,12 @@
+
+function info(name, breed, price) {
+  alert(name + ',' + ' ' + breed + ',' + ' ' + price);
+  }
+
+
+let total = 0;
+
+function carttot(price) {

```

Comment:
Small thing, but we generally use camel casing with names. So you'd want to do `function cartTot()`